libfwupdplugin/fu-dpaux-device.c

```diff
@@ -70,11 +70,25 @@ fu_dpaux_device_probe(FuDevice *device, GError **error)
 static gboolean
 fu_dpaux_device_setup(FuDevice *device, GError **error)
 {
+	FuContext *ctx = fu_device_get_context(device);
 	FuDpauxDevice *self = FU_DPAUX_DEVICE(device);
 	FuDpauxDevicePrivate *priv = GET_PRIVATE(self);
+	const gchar *name = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(self), "name", NULL);
 	guint8 buf[FU_STRUCT_DPAUX_DPCD_SIZE] = {0x0};
 	g_autoptr(GByteArray) st = NULL;
 
+	/* ignore all Framework FRANDGCP07 BIOS version 3.02 */
```

Comment:
I don't think you need to check both the name and GUID. I think just the guid is enough

Reply:
What about the other dp aux devices on the system? e.g. If i plug in a Lenovo tbt4 dock?

Reply:
I would expect this issue happens on all dp aux devices on the laptop, so should block all dp aux by hwid (or something else).

Reply:
Ohh, I thought it might be specific after https://github.com/fwupd/fwupd/issues/6308#issuecomment-1805778307

Reply:
It's probably continent upon what's plugged in to that DP port at the time.

Reply:
I think that's true for the DPCD, but:

    drivers/gpu/drm/amd/display/amdgpu_dm/amdgpu_dm_mst_types.c:            kasprintf(GFP_KERNEL, "AMDGPU DM aux hw bus %d",